app/models/piece.rb

```diff
@@ -93,4 +93,20 @@ def remains_on_board?(to_x:, to_y:)
       false
     end
   end
+
+  def successful_capture(to_x:, to_y:)
+    return false if obstructed_diagonally?(to_x:, to_y:)
+    return false if obstructed_vertically?(to_x:, to_y:)
+    return false if obstructed_horizontally?(to_x:, to_y:)
+    current_color = current_piece.color
```

Comment:
@asmentko What you're saying makes sense to me, seems like a good solution, Andrew. So in **game.rb**, below the two `initialize_board_` methods, we would add a `current_color` method to keep track of whose turn it is? Sounds good . . . will you create a new Trello card for this task and delete the one for the `current_piece` helper method?